src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const DISCOUNT_LONG_TERM = 50;
+  const SHORT_TERM = 3;
+  const DISCOUNT_SHORT_TERM = 20;
+
+  let basePrice = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return basePrice - DISCOUNT_LONG_TERM;
+  }
+
+  if (days >= SHORT_TERM) {
+    return basePrice - DISCOUNT_SHORT_TERM;
+  }
+
+  return PRICE_PER_DAY * days;
```

Comment:
You could return `basePrice` from line 13  instead of returning  `PRICE_PER_DAY * days` on line 23